packages/eslint-config-godaddy-es5/package.json

```diff
@@ -27,7 +27,8 @@
     "eslint-godaddy-es5-react": "bin/eslint-godaddy-es5-react"
   },
   "dependencies": {
-    "eslint-config-godaddy": "^3.0.0"
+    "eslint-config-godaddy": "^3.0.0",
+    "eslint-config-godaddy-react": "^5.0.0"
```

Comment:
I suggest we eliminate this es5 package; if someone really needs to code in ES5 they can use an old release.

Reply:
I'm also in favor of this tagging @godaddy/javascript for additional feedback on this